java/org/apache/tomcat/websocket/Constants.java

```diff
@@ -114,6 +114,22 @@
     // Milliseconds so this is 20 seconds
     public static final long DEFAULT_BLOCKING_SEND_TIMEOUT = 20 * 1000;
 
+    // Configuration for idle read timeout on websocket channel
+    public static final String WS_READ_IDLE_TIMEOUT =
+            "org.apache.tomcat.websocket.WS_READ_IDLE_TIMEOUT";
+
+    // Configuration for idle write timeout on websocket channel
+    public static final String WS_WRITE_IDLE_TIMEOUT =
+            "org.apache.tomcat.websocket.WS_WRITE_IDLE_TIMEOUT";
+
+    // Configuration for idle write timeout on websocket channel
```

Comment:
This comment is exactly the same as for the previous constant

Reply:
What is the purpose of the following two constants ? They are not used below.

Reply:
These were part of initial draft. 
I intended to remove them. 
Will do in next commit.